document/core/exec/numerics.rst

```diff
@@ -114,6 +114,10 @@ Floating-Point
 
 where :math:`M = \significand(N)` and :math:`E = \exponent(N)`.
 
+Fixed-width SIMD
+................
+
+:ref:`Fixed-width SIMD values <syntax-simd>` can be used to represent different types of packed data. The format of the this data is :math:`\{t\}\{lwidth\}\times \{lanes\}` and is interpreted as a packed SIMD value of :math:`lanes`number of values of type :math:`t`, which are either :ref:`integers <syntax-int>` or :ref:`floating-point <syntax-float>` numbers of width :math:`lwidth`.
```

Comment:
```suggestion
:ref:`Fixed-width SIMD values <syntax-simd>` can be used to represent different types of packed data. The format of this data is :math:`\{t\}\{\X{lwidth}\}\times \{\X{lanes}\}` and is interpreted as a packed SIMD value of :math:`\X{lanes}`number of values of type :math:`t`, which are either :ref:`integers <syntax-int>` or :ref:`floating-point <syntax-float>` numbers of width :math:`\X{lwidth}`.
```
(You need \X to typeset multi-letter var names correctly.)
Also style nit: At most one sentence per line.

However, my real question is that I don't understand the syntax you are suggesting. This ought to define the format for representing SIMD *values*, but the syntax doesn't seem to describe any actual values.

Reply:
Will be addressing other comments as well, but to address the general issue of syntax which is comes up in other review comments as well is that this PR is dependent on #193 where I've added some notion of a SIMD type, and some basic text for SIMD values, but it's unclear to me where the right place for the different representations is, I'd like to add some expressiveness for the different representations so this can reference that more accurately, and it will be easier to differentiate different operations that are applicable to different types.